services/harmony/app/util/aggregation-batch.ts

```diff
@@ -113,7 +113,7 @@ Promise<number[]> {
       outputItemSizes = outputItemSizes.concat(sizes);
     }
   }
-  return outputItemSizes;
+  return outputItemSizes.map(Math.floor);
```

Comment:
We convert `number` to `integer`. Are we not afraid of losing precision?

Reply:
The item sizes are supposed to be the number of bytes, which should be an integer

Reply:
This is a fix for cases where a weird rounding error was giving us values like 97.0000000001, which were breaking the dB insertion